networking/dedicated-federated-access.adoc

```diff
@@ -0,0 +1,59 @@
+[id="dedicated-federated-access"]
+= Federated Networking Access
+include::modules/common-attributes.adoc[]
+:context: dedicated-federated-access
+
+toc::[]
+
+Federated networking access allows Customer Portal Organization Admins and cluster owners to enable an Amazon Web Services (AWS) Identity and Access Management (IAM) users federated access to the AWS Console for their {product-title} cluster. Admins may select between Network Management or Read-only access options.
```

Comment:
```suggestion
Federated Access allows Customer Portal Organization Admins and cluster owners to enable an Amazon Web Services (AWS) Identity and Access Management (IAM) users federated access to the AWS Console for their {product-title} cluster. Admins may select between Network Management or Read-only access options.
```

"Federated Access" isn't the official name just yet I don't believe. It makes sense to me but the BU need to figure this out.